server/backend/database/memory/database.go

```diff
@@ -563,7 +563,12 @@ func (d *DB) findDeactivateCandidatesPerProject(
 	txn := d.db.Txn(false)
 	defer txn.Abort()
 
-	offset := gotime.Now().Add(-project.ClientDeactivateThreshold)
+	clientDeactivateThreshold, err := project.GetClientDeactivateThresholdAsTimeDuration()
```

Comment:
https://go.dev/doc/effective_go#Getters

```suggestion
	clientDeactivateThreshold, err := project.ClientDeactivateThresholdAsTimeDuration()
```

Reply:
Thank you for letting me know about Effective Go's getter name style.
Changed to `project.ClientDeactivateThresholdAsTimeDuration()`